ui/imports/shared/views/chat/ProfileContextMenu.qml

```diff
@@ -148,8 +148,9 @@ StatusMenu {
     SendContactRequestMenuItem {
         id: sendContactRequestMenuItem
         objectName: "sendContactRequest_StatusItem"
-        enabled: !root.isMe && !root.isContact
-                                && !root.isBlockedContact && !root.hasPendingContactRequest && !root.isBridgedAccount
+        enabled: !root.isMe && !root.isContact && !root.isBlockedContact
+                 && (contactDetails.contactRequestState === Constants.ContactRequestState.None || contactDetails.contactRequestState === Constants.ContactRequestState.Dismissed)
+                 && !root.isBridgedAccount
```

Comment:
`isBridgedAccount` means a discord account?

Reply:
Yeah, or other possible bridge in the future